spec/lib/twingly/url_spec.rb

```diff
@@ -171,6 +171,11 @@ def valid_urls
     it { is_expected.to eq("co.uk") }
   end
 
+  describe "#ttld" do
+    subject { url.ttld }
+    it { is_expected.to eq("uk") }
```

Comment:
Maybe me being a 🐔  but I think I'd like a test for `#ttld` when we have a normal domain as well, like a simple `.com`. I am thinking to expose problems in the way we split the `#tld`.